test/api_client/index.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright © 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import APIClient from 'api_client';
+
+describe('api client', () => {
+	describe('exports', () => {
+		it('should have APIClient as a function', () => {
+			return expect(APIClient).to.be.a('function');
```

Comment:
We might want to check if it's a constructor? I.e. capable of being instantiated.

Reply:
I thought all those tests should be covered here and following: https://github.com/LiskHQ/lisk-js/blob/development/test/api_client/api_client.js#L71